cmd/support_archive/builder.go

```diff
@@ -220,3 +229,17 @@ func printCopyCommand(log logd.Logger, tarballToStdout bool, tarFileName string)
 			podNamespace, podName, tarFileName, tarFileName)
 	}
 }
+
+func isSupportabilityEnabled(ctx context.Context, clientSet *kubernetes.Clientset) (bool, error) {
+	configMaps := clientSet.CoreV1().ConfigMaps(namespaceFlagValue)
+	options := metav1.GetOptions{}
+
+	configMap, err := configMaps.Get(ctx, installConfigMap, options)
+	if err != nil {
+		return false, err
+	}
+
+	supportabilityEnabled := strings.ToLower(configMap.Data[supportabilityField]) == "true"
+
+	return supportabilityEnabled, nil
+}
```

Comment:
this is not how you meant to use the `install-config`

add it to the deployment of the operator [just like the webhook](https://github.com/Dynatrace/dynatrace-operator/blob/8dea665463fb72c4d2eb36db74df68951f5432b3/config/helm/chart/default/templates/Common/webhook/deployment-webhook.yaml#L90-L92)
then use `GetModules` ([code](https://github.com/Dynatrace/dynatrace-operator/blob/8dea665463fb72c4d2eb36db74df68951f5432b3/pkg/util/installconfig/modules.go#L42))
- also pls add your `supportability` field to the [installconfig package](https://github.com/Dynatrace/dynatrace-operator/blob/8dea665463fb72c4d2eb36db74df68951f5432b3/pkg/util/installconfig/modules.go#L1)

Reply:
My fault, I made a mistake reading env vars. Fixed